pkg/util/util.go

```diff
@@ -29,6 +29,8 @@ import (
 
 const (
 	blockdevFileName = "/usr/sbin/blockdev"
+	// DefaultAlignBlockSize is the alignment size we use to align disk images.
```

Comment:
Since this is our const and not some well defined standard it would be nice to have some very short info on "why 1Mi". Unless you thin it is obvious.

Reply:
Well since you had to ask it is not obvious, but 1Mi is a multiple of all known HW block sizes 512/4k/8k/32k/64k thus it will align on all of them.